public/static/partials/user_host_options.html

```diff
@@ -79,12 +79,12 @@
         <textarea id="input-key-val" name="pubkey" placeholder="Enter public key here..." required key-base-valid ng-model="selectedKey.key"></textarea>
       </p>
       <p class="checkbox">
-        <input type="checkbox" id="input-key-chk" ng-click="toggleSaveKey()">&nbsp;&nbsp;Save public key&nbsp;&nbsp;</input>
-        <input type="text" id="input-key-name" name="keyname" ng-model="selectedKey.name" ng-disabled="!saveKey"; placeholder="mykey.pub" key-name-unique></input>
+        <input type="checkbox" id="input-key-chk" ng-click="toggleSaveKey()">&nbsp;&nbsp;Save public key&nbsp;&nbsp;
```

Comment:
Did you intentionally remove these end tags? This isn't valid HTML anymore

Reply:
According to [the internet](https://stackoverflow.com/questions/13232121/closing-html-input-tag-issue), you aren't actually supposed to have closing tags for input. I added the self-closing tag `/>` to make it clearer.